README.md

```diff
@@ -26,59 +26,78 @@ disable this feature, run with this environment variable:
-NODE_ROOT="${HOME}/usr/local/nodejs"
+Debian, by default, does not give your primary user account sudo access. Most of our scripts are designed assuming you will have sudo access so is most straightforward to simply add yourself to the sudoers file. You can do this with the following commands.
 
-# support x86 or ARM (e.g. Apple Silicon):
-ARCH=$(uname -m | sed 's/x86_/x/' | sed 's/aarch/arm/')
+```sh
+su - # this will prompt for the root password
+usermod -aG sudo <USERNAME> # use your user account username
+```
+Restart your machine for the changes to take effect. You can verify it worked after restart by entering `sudo whoami` in the terminal and you should see `root`. 
 
-# download and install:
-mkdir -p "$NODE_ROOT"
-curl -sLo- "https://nodejs.org/dist/v12.19.0/node-v12.19.0-linux-${ARCH}.tar.gz" | \
-    tar xz --strip-components 1 -C "${NODE_ROOT}"
+Next install git, and create an SSH key following the [github guide](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent). You will need to add the key to your github account. Then clone the vxsuite repositories. 
 
-# configure your shell; this assumes bash:
-echo "export PATH=\$PATH:${NODE_ROOT}/bin" >> ~/.bashrc
-export PATH="${PATH}:${NODE_ROOT}/bin"
-node -v # should print "v12.19.0"
+```sh
+sudo apt update
+sudo apt install -y git
+echo "export PATH=$PATH:/usr/sbin" >> ~/.bashrc # Only for debian, add sbin to your path
+ssh-keygen -t ed25519 -C "your_github_email@example.com" # Save to the default location, and chose a passphrase
+eval "$(ssh-agent -s)" # start the ssh agent
+ssh-add ~/.ssh/id_ed25519 # add your ssh key to the agent
+cat ~/.ssh/id_ed25519.pub # Copy the public key and add it to your github account
+ssh -T git@github.com # This should return `Hi username! You've successfully authenticated, but GitHub does not provide shell access.
+mkdir code
+cd code
+git clone git@github.com:votingworks/vxsuite.git
+
+# If you are doing a lot of development in vxsuite you will likely eventually need the following repos. 
+# kiosk-browser is an electron-based browser where our apps run in production.
+git clone git@github.com:votingworks/kiosk-browser.git
+# vxsuite-complete-system packages vxsuite and kiosk-browser for running in production with various setup scripts for production machines. If you want to test your code through kiosk-browser without needing to develop on kiosk-browser it is recommended you run kiosk-browser through the instructions in this repo.
+git clone git@github.com:votingworks/vxsuite-complete-system.git
+```
 
-# install pnpm:
-npm i -g pnpm@5
-pnpm -v # should print "v5.x.x"
+Once you finish setting up your VM before you start develop we recommend you also set up [GPG keys](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key) with your github account, but that is not necessary to do immediately while setting up.
```

Comment:
my two cents is yes, we should (even though I don't currently have it setup :grimacing:)

Reply:
Yeah, agreed, we should do this, and i haven't done it either.

Reply:
I've had this set up since around the time I started at VotingWorks. It's a little annoying because the tooling is not easy to work with. But I'm sure we can find a way to make it not suck.